### DISCUSSION TITLE
Add the possibility to stretch the sidebar

### DISCUSSION THREAD

**Discussion by JulienLecoq** [supabase/supabase#12706] [Feature Requests]
On the picture bellow, on the sidebar on the left, you can see that the table name is cropped. When you have multiple tables with similar name (the start of the name), you can't differentiate the tables visually. You have to put your cursor on the table name and wait something like 2/3 seconds for the full table name to appears in a popup which is annoying.

<img width="1728" alt="screenshot_supabase_ui" src="https://user-images.githubusercontent.com/54070985/221684626-460b25e4-16b6-4938-aeb7-1a9b7c7b3987.png">

I would like to have the possibility to stretch the sidebar as some website/application does, for example the prontonmail website allows to do that. That would allows to see the full table name without having to do the manipulation I mentioned earlier. (while stretching, the "bar" of the sidebar becomes pink on my Protonmail instance)

<img width="1725" alt="proton_ui" src="https://user-images.githubusercontent.com/54070985/221684813-e4db4640-c359-4623-8c51-88a9b7853fbe.png">

**Comment by ioucyf** [comment]
Very true, I experience the same issue often, and always go to the DevTools to change the css rules to "fix" it.
